src/effects/VST3/VST3Effect.cpp

```diff
@@ -365,6 +427,14 @@ bool VST3Effect::SaveSettings(
    return true;
 }
 
+
+EffectSettings VST3Effect::MakeSettings() const
+{
+   auto result = EffectSettings::Make<VST3EffectSettings>();
+   return result;
+}
```

Comment:
If you define `MakeSettings`, also define an efficient `CopySettingsContents`.

But make those definitions only very late, when all else is ready for the final switch to statelessness.  So, not in this first stage PR.